cime_config/machines/cmake_macros/oneapi-ifxgpu_aurora.cmake

```diff
@@ -3,5 +3,6 @@ string(APPEND CMAKE_EXE_LINKER_FLAGS " -lmkl_intel_lp64 -lmkl_sequential -lmkl_c
 if (compile_threaded)
   string(APPEND CMAKE_EXE_LINKER_FLAGS " -fiopenmp -fopenmp-targets=spir64")
 endif()
-string(APPEND SYCL_FLAGS " -\-intel -fsycl -fsycl-targets=spir64_gen -mlong-double-64 -Xsycl-target-backend \"-device 12.60.7\"")
+string(APPEND SYCL_FLAGS " -\-intel -fsycl -fsycl-targets=spir64_gen -mlong-double-64 ")
+string(APPEND OMEGA_SYCL_EXE_LINKER_FLAGS " -Xsycl-target-backend \"-device 12.60.7\" ")
```

Comment:
> I think we’d prefer not to change the E3SM machine file in an Omega PR. This change would be better suited for an E3SM PR, and we can downstream the change to Omega afterward.

This change is the same as in the upstream-E3SM. When a more recent E3SM-master is merged into `develop`, this change will disappear.

Reply:
Thanks!